bin/adhoc-scripts/mongoInit.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to connect to a MongoDB instance based on a set of parameters given at runtime
+Usage:
+      source /data/srv/current/apps/reqmgr2ms/etc/profile.d/init.sh
+      python3 -i -- mongoInit.py -c $WMCORE_SERVICE_CONFIG/reqmgr2ms-output/config-output.py
+"""
+
+import sys
+import logging
+import argparse
+
+from WMCore.Database.MongoDB import MongoDB
+
+from WMCore.Configuration import loadConfigurationFile
+
+if __name__ == '__main__':
+
+    FORMAT = "%(asctime)s:%(levelname)s:%(module)s:%(funcName)s(): %(message)s"
+    logging.basicConfig(stream=sys.stdout, format=FORMAT, level=logging.INFO)
+    logger = logging.getLogger(__name__)
+
+    parser = argparse.ArgumentParser(
+        prog='MongoInit',
+        formatter_class=argparse.RawTextHelpFormatter,
+        description=__doc__)
+
+    parser.add_argument('-c', '--config', required=True,
+                        help="""\
+                        The path to MSCONFIG to be used, e.g.
+                        for production: /data/srv/current/config/reqmgr2ms/config-output.py
+                        """)
+    args = parser.parse_args()
+
+    logger.info("Loading configFile: %s", args.config)
+    config = loadConfigurationFile(args.config)
+
+    msConfig = config.section_('views').section_('data').dictionary_()
+
+    msConfig.setdefault("mongoDBUrl", 'mongodb://localhost')
+    msConfig.setdefault("mongoDBPort", 27017)
+    msConfig.setdefault("mongoDB", 'msUnmergedDB')
+    msConfig.setdefault("mongoDBRetryCount", 3)
+    msConfig.setdefault("mongoDBReplicaset", None)
+    msConfig.setdefault("mockMongoDB", False)
+    msConfig.setdefault("collection", None)
```

Comment:
Because the set of collections in the micro service are dependent of the type of the microservice and are created with the service object instance, while this script here is intended to work standalone for creating a MonogoDB connection instance so that it can be later used in interactive mode with `ipython` or similar.

Reply:
this script is not mandatory to be used only with a service configuration file 
I use it in interactive session on a regular basis by just importing it in `ipython` and then pointing to the dtabase/collection I need.